src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -153,12 +153,13 @@ private void changeCheckInServicePointId(UUID servicePointId) {
     write(representation, "checkinServicePointId", servicePointId);
   }
 
-  private void changeItemStatusToDeclaredLost() {
+  public Loan changeItemStatus(ItemStatus itemStatus) {
     Item item = getItem();
     if (item != null) {
-      item.changeStatus(ItemStatus.DECLARED_LOST);
+      item.changeStatus(itemStatus);
     }
-    changeItemStatus(ItemStatus.DECLARED_LOST.getValue());
+    changeItemStatus(itemStatus.getValue());
```

Comment:
Now we have two `changeItemStatus` methods:
* `changeItemStatus(ItemStatus);`
* `changeItemStatus(String);`

Both of them are public, but seems each of them has a bit different logic - `changeItemStatus(ItemStatus)` updates item and loan vs `changeItemStatus(String)` updates loan only. 

Could you please rename them or use consistent logic, or declare `changeItemStatus(String)` as private?